app/views/users/_form.html.haml

```diff
@@ -0,0 +1,49 @@
+= form_for(@user) do |f|
+  - if @user.errors.any?
+    #error_explanation
+      %h2= "#{pluralize(@user.errors.count, "error")} prohibited this user from being saved:"
+      %ul
+        - @user.errors.full_messages.each do |msg|
+          %li= msg
+
+  %br
+  .field
+    = f.label :first_name
+    = f.text_field :first_name, :class => "form-control", :required => true
+  %br
+  .field
+    = f.label :last_name
+    = f.text_field :last_name, :class => "form-control", :required => true
+  %br
+  .field
+    = f.label :email
+    = f.email_field :email, :class => "form-control", :required => true
+  %br
+  .field
+    = f.label :gender
+    %select{:name=>"user[gender]", :id=> "user_gender"}
+      %option{:value=>"male"}Male
+      %option{:value=>"female"}Female
+  %br
+  .field
+    = f.label :status
+    %select{:name=>"user[status]", :id=> "user_status"}
+      %option{:value=>"shaved"}Shaved
+      %option{:value=>"massaged_and_trimmed"}Massaged and trimmed
+      %option{:value=>"trimmed"}Trimmed
+      %option{:value=>"unshaved"}Unshaved
+      %option{:value=>"not_applicable"}Not Applicable
+
+  %br
+  .actions
+    = f.submit 'Save', :class => "btn btn-success"
+
+  %br  
+
+
+%script{:type => "text/javascript"}
+  $(function() {
```

Comment:
The same helper will "help" you with setting selected values. You don't need JS for this.